plugins/web/opentelemetry-instrumentation-user-interaction/test/userInteraction.test.ts

```diff
@@ -55,6 +60,25 @@ describe('UserInteractionInstrumentation', () => {
     let dummySpanExporter: DummySpanExporter;
     let exportSpy: sinon.SinonSpy;
     let requests: sinon.SinonFakeXMLHttpRequest[] = [];
+
+    const registerInstrumentation = (
```

Comment:
Nit: consider changing name as it may be confused at first look with `registerInstrumentations`